### PR TITLE
"Never for this round" ghost poll generalization

### PR DESCRIPTION
**Pull request by AnturK**
Adds it to xenobio sentience potions

Going to replace old ones tommorow, so this is wip

### CODE REVIEW

**Comment by MrStonedOne** [comment]
The biggest issue is how do we let players know the scope of "never for this round" they might want that info.

File: code/_globalvars/lists/poll_ignore.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+//Each lists stores ckeys for "Never for this round" option category
+
+var/list/sentience_potion_ignore = list() //sentience potion query
```
Comment (RemieRichards) [inline code/_globalvars/lists/poll_ignore.dm]: why isn't this just a jagged assoc list?

Comment (AnturK) [inline code/_globalvars/lists/poll_ignore.dm]: Yeah, that probably gonna be cleaner. Will swap in a moment.

Comment (RemieRichards) [inline code/_globalvars/lists/poll_ignore.dm]: Plus then you can use the Assoc keys as category names in some form of debug view:
```
for(var/category in role_ignores)
    var/list/L = role_ignores[category]
    if(L && L.len)
        sometext += "[category]:<br>"
        for(var/ignorer in L)
            sometext += "[ignorner]<br>"
```

File: code/_globalvars/lists/poll_ignore.dm [hunk 2]
```diff
@@ -0,0 +1,6 @@
+//Each lists stores ckeys for "Never for this round" option category
+
+#define POLL_IGNORE_PAI "pai"
+#define POLL_IGNORE_SENTIENCE_POTION "sentience_potion"
+
+var/list/poll_ignore = list(POLL_IGNORE_PAI = list(),POLL_IGNORE_SENTIENCE_POTION = list())
```
Comment (RemieRichards) [inline code/_globalvars/lists/poll_ignore.dm]: you should lazy init these.

**Pull request merged**